src/runtime/gecko/util.js

```diff
@@ -0,0 +1,41 @@
+export { documentReady, getResourceLinks, getMeta } from '../web/util';
+import { getLangRevisionMap, getLangRevisionTuple } from '../web/util';
+
+export function getXULResourceLinks(doc) {
+  return Array.prototype.map.call(
+    doc.querySelectorAll('messagebundle'),
```

Comment:
Have you considered using `<html:link>` elements instead of `<messagebundle>`?  Right now it looks like `gteResourceLinks` is the only difference between `l20n-xul` and `l20n-html` and I'm wondering if we could actually just use one.